ci/fih-tests_run.sh

```diff
@@ -19,7 +19,7 @@ set -e
 pushd .. &&\
    git clone https://git.trustedfirmware.org/TF-M/trusted-firmware-m.git &&\
    pushd trusted-firmware-m &&\
-   git checkout TF-Mv1.7.0 &&\
+   git fetch https://review.trustedfirmware.org/TF-M/trusted-firmware-m refs/changes/09/18309/2 && git checkout FETCH_HEAD &&\
```

Comment:
Will this link remain valid?

Reply:
I don't think so. Its only a temporary workaround because of the dependencies between MCUboot and TF-M, ask @tamasban  for details.

Reply:
Not sure what happens with the link if the change got merged to TF-M.

I think this flow could work:
- First, merge this PR with the link to the change on review in TF-M repo.
- Then I can create another change in TF-M with exact same content as the current change + update the MCUboot version (which includes the FIH enhancement) in TF-M repo. That can be merged and in this way, the current link remains valid.
- Finally, a new PR in MCUboot to update the TF-M version in the FIH CI job to be aligned.
- Current WIPchnage in TF-M can be abandoned.